### ISSUE TITLE
Accessibility - Submit Screen - Poor Color Contrast for Drop-down List

### ISSUE THREAD

**Issue by jfc3-dol** [USDepartmentofLabor/OCIO-DOLSafety-iOS#47]
The color contrast between the drop-down list text ( #1F1F1F ) and the background ( #7F7F7F ) is not enough. Please see screenshot. The color contrast ratio is 4.12:1 and needs to be 4.5:1 or higher to pass. The color contrast ratio came from the darker text, so the lighter-colored text is even lower.

Please see the attached screenshot.

![safetynet - anonymous incident screen - color contrast of the of the dark gray text on the gray background](https://user-images.githubusercontent.com/10423487/61563446-f619e880-aa41-11e9-82a7-e25ed044dd46.png)

**Comment by aanthonyDOL** [comment]
Are you inferring that the drop-down list background color should be lighter than the text

**Comment by aanthonyDOL** [comment]
The Picker was removed.

**Comment by jfc3-dol** [comment]
Yes, I meant the background color of the drop-down list and the color of the text on it.

When the next version of the application comes out will make sure the drop-down list has been removed. 

What did you put in its place?
